src/lnp/java/teammates/lnp/cases/InstructorStudentCascadingUpdateLNPTest.java

```diff
@@ -0,0 +1,360 @@
+                                        .withCourseId(COURSE_ID)
+                                        .withFeedbackResponseId(String.valueOf(i))
+                                        .withFeedbackSessionName(FEEDBACK_SESSION_NAME)
+                                        .withGiverSection(String.valueOf(j / NUM_STUDENTS_PER_SECTION))
+                                        .withCommentGiver(String.valueOf(j / NUM_STUDENTS_PER_SECTION))
+                                        .withCommentText(responseText)
+                                        .build());
+                    }
+                }
+
+                return feedbackResponseComments;
+            }
+
+            @Override
+            protected Map<String, InstructorAttributes> generateInstructors() {
+                Map<String, InstructorAttributes> instructors = new HashMap<>();
+
+                instructors.put(INSTRUCTOR_ID,
+                        InstructorAttributes.builder(COURSE_ID, INSTRUCTOR_EMAIL)
+                                .withGoogleId(INSTRUCTOR_ID)
+                                .withName(INSTRUCTOR_NAME)
+                                .withRole("Co-owner")
+                                .withIsDisplayedToStudents(true)
+                                .withDisplayedName("Co-owner")
+                                .withPrivileges(new InstructorPrivileges(
+                                        Const.InstructorPermissionRoleNames.INSTRUCTOR_PERMISSION_ROLE_COOWNER))
+                                .build()
+                );
+
+                return instructors;
+            }
+
+            @Override
+            public List<String> generateCsvHeaders() {
+                List<String> headers = new ArrayList<>();
+
+                headers.add("loginId");
+                headers.add("isAdmin");
+                headers.add("courseId");
+                headers.add("enrollData");
+
+                return headers;
+            }
+
+            @Override
+            public List<List<String>> generateCsvData() {
+                DataBundle dataBundle = loadDataBundle(getJsonDataPath());
+                List<List<String>> csvData = new ArrayList<>();
+
+                dataBundle.instructors.forEach((key, instructor) -> {
+                    List<String> csvRow = new ArrayList<>();
+
+                    csvRow.add(instructor.googleId);
+                    csvRow.add("yes");
+                    csvRow.add(instructor.courseId);
+
+                    // Create and add student enrollment data with a team number corresponding to each section number
+                    List<StudentsEnrollRequest.StudentEnrollRequest> enrollRequests = new ArrayList<>();
+                    int startIndex = csvTestDataIndex * NUM_STUDENTS_PER_SECTION;
+
+                    for (int i = startIndex; i < startIndex + NUM_STUDENTS_PER_SECTION; i++) {
+                        String name = instructor.name + ".Student" + (NUM_STUDENTS - i);
```

Comment:
Can clarify if the students have a name change here? From the above, the logic seems to generate students with name `STUDENT_NAME_PREFIX + i`?

Reply:
Yes they have a name change here to incur more updates

Reply:
For `test.app.domain`, I just want to have less code changes to `JMeterElements` class since it will affect every other LnP test

Reply:
I see! Thanks for the clarification.
On the domain part, maybe we should clarify with the rest. My personal take is just 1 url for simplicity but you do have a point. Let's stash that discussion.